src/graphql/schema/post.js

```diff
@@ -0,0 +1,82 @@
+const sharedPostInputFields = `
+  latest: String!
+  page: Int!
+  pageSize: Int!
+`;
+
+export default `
+  type Post {
+    id: ID!
+    title: String!
+    url: String!
+    image: String
+    ratio: Float
+    placeholder: String
+    publication_id: ID!
```

Comment:
The post object which returned from `generateFeed` doesn't have a `publication_id` property although the schema does contain it. IMO we should remove this field.